packages/plugins/plugin-preview/src/capabilities/react-surface.tsx

```diff
@@ -65,9 +65,11 @@ export default () =>
           [dispatch],
         );
         return (
-          <ContactCard subject={data.subject} onOrgClick={handleOrgClick} role={role}>
-            {role === 'popover' && <Surface role='related' data={data} />}
-          </ContactCard>
+          <CardContainer role={role}>
```

Comment:
any reason why this isn't within `ContactCard`?

Reply:
I gave this a lot of thought – one of the problems with any pattern that can be called "card" is where it gets its encapsulation, since that is a common pattern across the system. Encapsulated patterns might host a card, like a popover, so the card might not need to provide its own encapsulation.

We've already established a pattern of signaling surface situations through `role`, though since this concept diverges from the HTML attribute it seems most appropriate at first blush to handle that in pluginland.

I'm amenable to moving this or finding another way of tracking encapsulation, just don't want to optimize overly early, hence the current proposal.

Reply:
Ah, turns out we need this for other plugins so I’m already going to move it upstream.

Reply:
I didn't necessarily mean removing `CardContainer` but questioning why `<CardContainer role={role}>` wasn't a part of the `ContactCard`, which I would still ask even if its factor out.